Directory.Build.props

```diff
@@ -8,6 +8,15 @@
     <_OutputPath>$(MSBuildThisFileDirectory)bin\Build$(Configuration)\</_OutputPath>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <ProduceReferenceAssemblyInOutDir>true</ProduceReferenceAssemblyInOutDir>
+    
+    <!-- 
+      Workaround for https://github.com/NuGet/Home/issues/6461 (VSWin Only)
+      Even though we don't build NuGet packages, it still attempts to build the NuGet package name, which includes 
+      the assembly version if PackageVersion isn't specified. Because our assemblies have different versions
+      per-TFM, that causes a problem because the package can't have multiple versions in it.  
+      Set a dummy PackageVersion because we don't actually need it.
+    -->
+    <PackageVersion>1.0.0.0</PackageVersion>
```

Comment:
I guess we'd change this if we ever ship a NuGet package from this repo? Maybe wouldn't happen.

Reply:
I guess we could drop this hack when we no longer target classic.

Reply:
If (when?) we ever ship a NuGet package from this repo, it should presumably match the version of `Java.Interop.dll`, which will match the .NET version it's shipping with (6.0.0.0, 7.0.0.0, etc.).

Thus, perhaps we should set `$(PackageVersion)`=6.0.0.0?

Reply:
Given that we have no intentions of releasing a NuGet package from this repo, I suspect a `6.0.0.0` version will be just as obsolete as `1.0.0.0` if we ever do ship a package.  😁